spec/index.bs

```diff
@@ -3551,6 +3551,7 @@ dictionary GPUTextureCopyView {
     required GPUTexture texture;
     GPUIntegerCoordinate mipLevel = 0;
     GPUOrigin3D origin = {};
+    GPUTextureAspect aspect = "all";
```

Comment:
in light of Metal not allowing separate aspects in texture-texture copies, maybe `GPUTextureDataLayout` is a better place to put this?

Reply:
I think it still makes more sense on the GPUTextureCopyView and we should just validate against the cases that don't work in Metal.